fetch.bs

```diff
@@ -3284,15 +3271,9 @@ the request.
  <a for="fetch params">process response end-of-body</a> is <var>processResponseEndOfBody</var>, and
  <a for="fetch params">task destination</a> is <var>taskDestination</var>.
 
- <li>
-  <p>If <var>request</var>'s <a for=request>body</a> is a <a for=/>byte sequence</a>, then:
-
-  <ol>
-   <li><p>Let <var>body</var> and <var ignore>ignoreType</var> be the result of
-   <a for=BodyInit>safely extracting</a> <var>request</var>'s <a for=request>body</a>.
-
-   <li><p>Set <var>request</var>'s <a for=request>body</a> to <var>body</var>.
-  </ol>
+ <li><p>If <var>request</var>'s <a for=request>body</a> is a <a for=/>byte sequence</a>, then set
+ <var>request</var>'s <a for=request>body</a> to the first return value of
```

Comment:
"the first return value" seems pretty unusual; why not just "the result of"? Here and below.

Reply:
It returns two values. I'm aligning it with text elsewhere that uses this terminology.